src/cmd/tag.cpp

```diff
@@ -15,8 +16,8 @@ void Tag::setOptions(cxxopts::Options &opts) {
     .positional_help("[args]")
     .custom_help("tag [tag]")
     .add_options()
-    ("t,tag", "New tag", cxxopts::value<std::string>()->default_value(""));
-
+    ("t,tag", "New tag", cxxopts::value<std::string>()->default_value(""))
+    ("r,registry", "Registry", cxxopts::value<std::string>()->default_value(""));
```

Comment:
You shouldn't need two fields; a tag can include a registry URL (e.g. testhub1.dronedb.app/admin/dataset).

Reply:
I just fixed it. We still need the registry parameter when getting the tag